app/models/manageiq/providers/ibm_cloud/object_storage/storage_manager.rb

```diff
@@ -92,6 +89,10 @@ def self.params_for_create
     }.freeze
   end
 
+  def queue_name_for_ems_refresh
+    "object"
+  end
```

Comment:
Curious what you're trying to solve with this as this is almost certainly incorrect

Reply:
We had a problem that _queue_name_for_ems_refresh_ was non-existent for the object storages, so we had to define it explicitly in our class. I will check now if it has been solved in the parent manager.

Reply:
`app/models/manageiq/providers/storage_manager.rb:    delegate :queue_name_for_ems_refresh, :to => :parent_manager` should cover this for you

Reply:
Ok, sounds good, then I'll remove the method.

Reply:
If you didn't have a parent manager you would want `def queue_name_for_ems_refresh; queue_name: end`

Reply:
COS is a stand-alone manager, so it might exist without a parent unless the abstract parent manager that we wanted to achieve is already in place.

Reply:
:+1:  okay so you want this to be `queue_name` (which is what is in the base ExtManagementSystem) instead of `"object"`